lib/tests/EventLoopGroupTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require_once('tests.inc');
+
+final class EventLoopGroupTest extends PHPUnit_Framework_TestCase {
+
+    public function testLifetime() {
+        $elg = new EventLoopGroup(new EventLoopGroupOptions());
+        $elg = null;
```

Comment:
These 2 tests have no assert or verification. Unless the contructor/destructor horrible crashes or the declaration is missing this is never going to fail.

Reply:
that's correct. There's nothing to assert on. However, these tests found bugs in my EventLoopGroup/NativeResource code (forgot to release a reference, so PHP crashed)

Reply:
We have these same tests in every CRT, they are designed to ensure that references to native resources actually drop when we expect them to.